internal/cli/kraft/cloud/volume/template/remove/remove.go

```diff
@@ -0,0 +1,140 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2025, Unikraft GmbH and The KraftKit Authors.
+// Licensed under the BSD-3-Clause License (the "License").
+// You may not use this file except in compliance with the License.
+
+package remove
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/spf13/cobra"
+
+	kraftcloud "sdk.kraft.cloud"
+
+	"kraftkit.sh/cmdfactory"
+	"kraftkit.sh/config"
+	"kraftkit.sh/internal/cli/kraft/cloud/utils"
+	"kraftkit.sh/log"
+)
+
+type RemoveOptions struct {
+	Auth   *config.AuthConfig    `noattribute:"true"`
+	Client kraftcloud.KraftCloud `noattribute:"true"`
+	All    bool                  `long:"all" short:"a" usage:"Remove all templates"`
+	Metro  string                `noattribute:"true"`
+	Token  string                `noattribute:"true"`
+}
+
+// Remove a KraftCloud persistent volume.
+func Remove(ctx context.Context, opts *RemoveOptions, args ...string) error {
+	if opts == nil {
+		opts = &RemoveOptions{}
+	}
+
+	return opts.Run(ctx, args)
+}
+
+func NewCmd() *cobra.Command {
+	cmd, err := cmdfactory.New(&RemoveOptions{}, cobra.Command{
+		Short:   "Permanently delete volume template(s)",
+		Use:     "remove [FLAGS] [UUID|NAME [UUID|NAME]...]",
```

Comment:
```diff
- 		Use:     "remove [FLAGS] [UUID|NAME [UUID|NAME]...]",
+		Use:     "remove [FLAGS] UUID|NAME[,UUID|NAME...]",
```

Reply:
It is optional because you can use `--all` in which case no arguments should be provided